modules/metal-connection/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.5.4"
+  required_providers {
+    equinix = {
+      source  = "equinix/equinix"
+      version = ">= 1.20.0"
```

Comment:
This needs to be updated with the latest version of the TF provider after the merge of this PR: https://github.com/equinix/terraform-provider-equinix/pull/626/files#diff-703d45b76399eb49451408ebb9688abdad1236c9cfb95664c5044fe2488e19fb